tests/core/test_solve.py

```diff
@@ -552,7 +544,7 @@ def test_update_all_1():
 
 
 def test_broken_install():
-    specs = MatchSpec("pandas"), MatchSpec("python=2.7"), MatchSpec("numpy 1.6.*")
+    specs = MatchSpec("pandas=0.11.0=np16py27_1"), MatchSpec("python=2.7")
```

Comment:
We change numpy to be an implicit spec here.  Without doing this, conda will fix the inconsistent spec that we introduce later more aggressively, and our test won't work.